tests/test_simsetup.py

```diff
@@ -1081,7 +1089,48 @@ def test_param_select_redundant():
     assert uv_obj_red.Nbls < uv_obj_full.Nbls
 
 
-@pytest.mark.filterwarnings("ignore:Cannot check consistency of a string-mode BeamList")
+@pytest.mark.parametrize(
+    "order_dict",
+    [
+        None,
+        {"conjugation_convention": "v>0", "blt_order": ["ant1", "ant2"]},
+        {"conjugation_convention": "ant2<ant1", "blt_order": ["baseline"]},
+        {"conjugation_convention": "ant2<ant1", "blt_order": "baseline"},
+    ],
+)
+def test_param_ordering(order_dict):
+    param_filename = os.path.join(
+        SIM_DATA_PATH, "test_config", "obsparam_mwa_nocore.yaml"
+    )
+    param_dict = pyuvsim.simsetup._config_str_to_dict(param_filename)
+    uv_obj_orig = pyuvsim.initialize_uvdata_from_params(param_dict, return_beams=False)
+
+    param_dict2 = copy.deepcopy(param_dict)
+    if order_dict is None:
+        param_dict2.pop("ordering")
+        warn_str = (
+            "The default baseline conjugation convention has changed. In the past "
+            "it was 'ant2<ant1', it now defaults to 'ant1<ant2'. You can specify "
+            "the baseline conjugation convention in `obs_param` by setting the "
+            "obs_param['ordering']['conjugation_convention'] field. This warning "
+            "will go away in version 1.5."
+        )
+        warn_type = UserWarning
+    else:
+        warn_type = None
+        warn_str = ""
+        param_dict2["ordering"] = order_dict
+    with uvtest.check_warnings(warn_type, match=warn_str):
+        uv_obj2 = pyuvsim.initialize_uvdata_from_params(param_dict2, return_beams=False)
+
+    if order_dict is not None:
+        uv_obj2.reorder_blts(
+            order="time", minor_order="baseline", conj_convention="ant1<ant2"
+        )
```

Comment:
Is it prudent to check we are getting the correct ordering and convention before turning it back to match the orig object? Or do we do that somewhere else too?

Reply:
yes. I did it and realized that the logic wasn't quite right. Thanks!